packages/client/src/components/WelcomeBox/Heading/Heading.stories.js

```diff
@@ -0,0 +1,16 @@
+import { Heading } from './Heading.component';
```

Comment:
Related to the component feedback. Storybook is also about documentation and showing other developers which **reusable** components exist in the code base, but if this component isn't meant to be used in more places, maybe having a story for each part of the WelcomeBox becomes clutter in the Storybook UI? 
If I can't drop a component on it's own then maybe it shouldn't be made so visible?